bigtable/tableadmin/tableadmin.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to connect to Cloud Bigtable and run some basic operations.
+#     http://www.apache.org/licenses/LICENSE-2.0
+Prerequisites:
+- Create a Cloud Bigtable cluster.
+  https://cloud.google.com/bigtable/docs/creating-cluster
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+
+Operations performed:
+- Create bigtable.
+- List bigtables existing in current project, if any.
+- List metadata(Name) of created bigtable.
+- Create Column Families with different GC rules.
+    - GC Rules like: MaxAge, MaxVersions, Union, Intersection and Nested.
+- Delete bigtable.
+"""
+
+import argparse
+import datetime
+
+from google.cloud import bigtable
+
+
+def run_table_operations(project_id, instance_id, table_id):
+    ''' Create bigtable and perform different operations on it.

```

Comment:
Create a Bigtable table and perform basic table operations

Reply:
Done